bot.py

```diff
@@ -2137,6 +2165,90 @@ async def on_guild_channel_update(
                 f"Category {before.id} was updated but is not associated with a role in cache. This could be an error."
             )
 
+@bot.event
+async def on_guild_emojis_update(guild: discord.Guild, before: Sequence[discord.Emoji], after: Sequence[discord.Emoji]):
```

Comment:
If the bot uploaded the emoji, it should not trigger the event. Currently if the bot uploads the emoji to a server it still triggers an approval for itself.

Reply:
This can be determined by using `emoji.user` in the `after` or `before` list to determine which user uploaded a given emoji.